<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>2221</width>
    <height>920</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Terrain Generator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="m_MainWindowgridLayout">
    <item row="1" column="0">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>30</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="1">
     <widget class="QGroupBox" name="gridNoiseGroupBox">
      <property name="title">
       <string>Grid and Noise Controls         </string>
      </property>
      <widget class="QDoubleSpinBox" name="freqSpinBox">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>50</y>
         <width>66</width>
         <height>27</height>
        </rect>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
       <property name="value">
        <double>3.000000000000000</double>
       </property>
      </widget>
      <widget class="QSpinBox" name="octavesSpinBox">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>110</y>
         <width>45</width>
         <height>27</height>
        </rect>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>11</number>
       </property>
       <property name="value">
        <number>7</number>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>81</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string>Frequency</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>90</y>
         <width>63</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string>Octaves</string>
       </property>
      </widget>
      <widget class="QSlider" name="widthHorizontalSlider">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>250</y>
         <width>100</width>
         <height>20</height>
        </rect>
       </property>
       <property name="minimum">
        <number>50</number>
       </property>
       <property name="maximum">
        <number>500</number>
       </property>
       <property name="value">
        <number>300</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QSlider" name="depthVerticalSlider">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>150</y>
         <width>20</width>
         <height>100</height>
        </rect>
       </property>
       <property name="minimum">
        <number>50</number>
       </property>
       <property name="maximum">
        <number>500</number>
       </property>
       <property name="value">
        <number>300</number>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>220</y>
         <width>63</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string>W</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>190</y>
         <width>63</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string>D</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="ratioLockCheckBox">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>190</y>
         <width>91</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>Lock ratio</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="heightSpinBox">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>50</y>
         <width>45</width>
         <height>27</height>
        </rect>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>199</number>
       </property>
       <property name="value">
        <number>90</number>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>30</y>
         <width>63</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string>Height</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>280</y>
         <width>121</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>280</y>
         <width>51</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>W</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>280</y>
         <width>141</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string>to toggle wireframe</string>
       </property>
      </widget>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Erosion Controls</string>
      </property>
      <widget class="QDial" name="totalDropletsDial">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>110</y>
         <width>50</width>
         <height>64</height>
        </rect>
       </property>
       <property name="minimum">
        <number>1000</number>
       </property>
       <property name="maximum">
        <number>40000</number>
       </property>
       <property name="singleStep">
        <number>10</number>
       </property>
       <property name="pageStep">
        <number>1000</number>
       </property>
       <property name="value">
        <number>20000</number>
       </property>
       <property name="tracking">
        <bool>true</bool>
       </property>
       <property name="notchesVisible">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLCDNumber" name="totalDropletsLabel">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>130</y>
         <width>64</width>
         <height>23</height>
        </rect>
       </property>
       <property name="intValue" stdset="0">
        <number>20000</number>
       </property>
      </widget>
      <widget class="QPushButton" name="erodeButton">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>310</y>
         <width>84</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>Erode</string>
       </property>
      </widget>
      <widget class="QDial" name="lifetimeDial">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>170</y>
         <width>50</width>
         <height>64</height>
        </rect>
       </property>
       <property name="value">
        <number>30</number>
       </property>
       <property name="notchesVisible">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>110</y>
         <width>101</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string>Droplet Total</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>41</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>V</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>41</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>E</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>30</y>
         <width>151</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string>to toggle visualisation</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>70</y>
         <width>151</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string>to quick Erode </string>
       </property>
      </widget>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>170</y>
         <width>111</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string>Droplet Lifetime</string>
       </property>
      </widget>
      <widget class="QLCDNumber" name="lifetimeLabel">
       <property name="geometry">
        <rect>
         <x>73</x>
         <y>190</y>
         <width>61</width>
         <height>23</height>
        </rect>
       </property>
       <property name="intValue" stdset="0">
        <number>30</number>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>2221</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
